community-operators/opsmx-spinnaker-operator/1.20.3/opsmx-spinnaker-operator.v1.20.3.clusterserviceversion.yaml

```diff
@@ -164,5 +353,5 @@ spec:
   maturity: alpha
   provider:
     name: OpsMx
-  replaces: opsmx-spinnaker-operator.v1.17.4
+  replaces: opsmx-spinnaker-operator.v1.20.3
```

Comment:
@vkvamsiopsmx 1.20.3 is not replacing the older version, but itself. It is not possile.